packages/api/src/platforms/vtex/index.ts

```diff
@@ -1,12 +1,15 @@
+import type { Clients } from './clients'
 import { getClients } from './clients'
+import type { SearchArgs } from './clients/search'
```

Comment:
**curiosity:** Just checking, all these import changes were made when you added the "source.organizeImports": "explicit" in the vscode settings, right?

Reply:
Yes!